Sources/TuistCore/Models/PackageSettings.swift

```diff
@@ -18,28 +18,39 @@ public struct PackageSettings: Equatable, Codable {
     /// The custom project options for each project generated from a swift package
     public let projectOptions: [String: XcodeGraph.Project.Options]
 
+    /// A Boolean value that indicates whether the test targets of local swift packages are included in generated project.
+    ///
+    /// If only SPM project, test targets are always included regardless of the value of this property.
+    public var includeLocalPackageTestTargets: Bool
+
     /// Swift tools version of the parsed `Package.swift`
     public let swiftToolsVersion: Version
 
     /// Initializes a new `PackageSettings` instance.
     /// - Parameters:
     ///    - productTypes: The custom `Product` types to be used for SPM targets.
+    ///    - productDestinations: Custom destinations to be used for SPM products.
     ///    - baseSettings: The base settings to be used for targets generated from SwiftPackageManager
     ///    - targetSettings: The custom `SettingsDictionary` to be applied to denoted targets
     ///    - projectOptions: The custom project options for each project generated from a swift package
+    ///    - includeLocalPackageTestTargets: A Boolean value that indicates whether the test targets of
+    ///    local swift packages are included in generated project.
+    ///    - swiftToolsVersion: Swift tools version of the parsed `Package.swift`
```

Comment:
Thanks for adding the missing docs here 🙂

Reply:
Of course I should 🙌